.github/workflows/ci.yml

```diff
@@ -103,7 +92,7 @@ jobs:
       id: coverage
       if: always() && contains('success,failure', steps.test.outcome)
       # coverage reporting must use the same Python version used to produce coverage
-      run: tox -qe coverage$(echo '${{ matrix.python-version }}' | tr -dc '0-9')
+      run: tox -qe coverage$(tr -d "." <<< "${{ matrix.python-version }}")
```

Comment:
Won't `3.13-dev` cause a problem here?

Reply:
Yes....but that's a problem for whoever adds 3.13 support ;)

I'll restore the slightly more complicated `tr`